Lib/idlelib/idle_test/test_configdialog.py

```diff
@@ -666,16 +744,21 @@ def test_delete_custom(self):
         idleConf.userCfg['highlight'].SetOption(theme_name, 'name', 'value')
         highpage[theme_name] = {'option': 'True'}
 
+        theme_name2 = 'other theme'
+        idleConf.userCfg['highlight'].SetOption(theme_name2, 'name', 'value')
+        highpage[theme_name2] = {'option': 'False'}
+
```

Comment:
This and the corresponding additions for keys cover the failure of `if not item_list` and the `else` clause.  I suspect it could be done in fewer lines, but I hope to someday factor out the common code, simpfiy the UI, and redo the tests anyway.  So leave the additions alone.